lib/MultiReporters.js

```diff
@@ -63,10 +63,21 @@ function MultiReporters(runner, options) {
         );
     }
 }
-util.inherits(MultiReporters, Base)
+
+util.inherits(MultiReporters, Base);
 
 MultiReporters.CONFIG_FILE = '../config.json';
 
+MultiReporters.prototype.done = function (failures, fn) {
+    if (_.size(this._reporters) === 0) return;
+
+    this._reporters.forEach(function(reporter) {
+        if (typeof reporter.done === 'function') {
+            reporter.done(failures, fn);
```

Comment:
My test scenario was running xunit and mochawesome together with this change through grunt-mocha-test.

If we pass in the 'fn' argument here, it breaks mochawesome when run with xunit. If produces an empty index.html. If you run it again (without deleting the previous reports), it typically works. If you run mochawesome alone, it works.

My suspicion is the discrepancy may be a timing issue. Xunit executes 'fn' and kills the processing before mochawesome's done completes. I suppose we could classify it as a xunit bug, but since mocha is really only built for one reporter, I'm not so sure.

If we just pass 'function(){}' instead of 'fn', this scenario works. I'm not really sure of the potential issue of not passing 'fn' to each reporter.